packages/x-babel-config/index.js

```diff
@@ -1,4 +1,4 @@
-module.exports = (targets = []) => ({
+module.exports = (targets = [], modules = false) => ({
```

Comment:
when calling this function it isn't clear what the arguments are for. i'd suggest using an object:

```suggestion
module.exports = ({ targets = [], modules = false } = {}) => ({
```